drivers/veteran_confirmation/extensions/grda_warehouse/hud/client_extension.rb

```diff
@@ -0,0 +1,31 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module VeteranConfirmation::GrdaWarehouse
+end
+
+module VeteranConfirmation::GrdaWarehouse::Hud
+  module ClientExtension
+    extend ActiveSupport::Concern
+
+    MIN_DAYS = 7.days
+
+    def va_check_enabled?
```

Comment:
I think this should return false if `veteran_override` so we don't check again if we have a prior positive.